### DISCUSSION TITLE
Changing llm models.

### DISCUSSION THREAD

**Discussion by sidharth-14** [langchain-ai/langserve#429] [Q&A]
How can I change the llm models after I have deployed the chain using langserve. I want to change the model based on the input on the langserve playground. Is this achievable?

**Comment by rever1and** [comment]
using configurable?

**Reply by sidharth-14** [reply]
Yes

**Reply by eyurtsev** [reply]
If you're using configurable, the playground should show a configuration option for each registered LLM.

Use configurable alternatives for this purpose.

**Reply by sidharth-14** [reply]
Thank you @eyurtsev
